src/test/java/com/mojang/brigadier/StringReaderTest.java

```diff
@@ -527,4 +529,24 @@ public void readBoolean_none() throws Exception {
             assertThat(ex.getCursor(), is(0));
         }
     }
+
+    @Test
+    public void isAllowedInUnquotedStringInput2OutputTrue() {
+      assertTrue(StringReader.isAllowedInUnquotedString('2'));
+    }
+
+    @Test
+    public void isAllowedInUnquotedStringInputNotNullOutputFalse() {
+      assertFalse(StringReader.isAllowedInUnquotedString('\u0000'));
+    }
+
+    @Test
+    public void isAllowedInUnquotedStringInputNotNullOutputFalse2() {
+      assertFalse(StringReader.isAllowedInUnquotedString(':'));
+    }
+
+    @Test
+    public void isAllowedInUnquotedStringInputrOutputTrue() {
+      assertTrue(StringReader.isAllowedInUnquotedString('r'));
+    }
 }
```

Comment:
Missing newline at end of file

Reply:
The missing newline is present in the original file.